sigopt/local_run_context.py

```diff
@@ -0,0 +1,41 @@
+from .run_context import BaseRunContext
+import copy
+
+class LocalRunContext(BaseRunContext):
+  def __init__(self, **kwargs):
+    self.run = copy.deepcopy(kwargs) if kwargs else {}
+
+  def get(self, name=None, type=dict):
```

Comment:
this could be enforced by lint, but please don't override the builtin `type`

Reply:
Fixed.